pkg/parser/parser_test.go

```diff
@@ -0,0 +1,86 @@
+package parser
+
+import (
+	"testing"
+
+	"github.com/tjgurwara99/monkey-go/pkg/ast"
+	"github.com/tjgurwara99/monkey-go/pkg/lexer"
+)
+
+func TestLetStatements(t *testing.T) {
+	input := `
+		let x = 5;
+		let y = 10;
+		let foobar = 838383;
```

Comment:
maybe also add the functionality of `func` expressions? As `func` produces a value that could be bound to a variable in monkey.

Reply:
They are not parsed yet, but will add them later.